src/main/java/uk/gov/hmcts/cmc/claimstore/services/pilotcourt/PilotCourtService.java

```diff
@@ -64,14 +66,24 @@ public void init() {
         return pilotCourts.keySet();
     }
 
+    public Set<String> getPilotCourtIds(Pilot pilot) {
```

Comment:
Isn't this making assumptions about how the consumer wants to handle the court data? I would expect the Pilot Court Service to return collections of Courts and let the consumer continue processing as they wish. (Also, since it also has methods returning Hearing Courts, wouldn't this class be better named as just a Court Service?)